vignettes/TADAGeospatialFunctions.Rmd

```diff
@@ -0,0 +1,236 @@
+---
+title: "TADA Geospatial Functions"
+format: html
+editor: visual
+author: "TADA Team"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\usepackage[utf8]{inputenc}
+  %\VignetteIndexEntry{TADA Geospatial Functions}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+  markdown: 
+    wrap: 72
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  eval = FALSE,
+  fig.width = 8,
+  fig.height = 8
+)
+```
+
+## Welcome!
+
+Thank you for your interest in Tools for Automated Data Analysis (TADA). TADA is an open-source tool set built in the R programming language. This [RMarkdown](https://bookdown.org/yihui/rmarkdown/) document walks users through how to download the TADA R package from GitHub, access and parameterize several important functions, and create basic visualizations with a sample data set. The sample data set contains data from one week from
```

Comment:
The paragraphs at the start of the vignette don't have manual line breaks but the ones later on seem to. Should they all be manual?

Reply:
I think it's an rmarkdown wrap:72 issue. I think I've resolved it!